docs/contribute.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Primer
+
+There are several ways you can contribute to and participate in Primer:
+
+- Participate in discussions
+- Submit an issue
+- Improve the documentation
+- Close issues
+- Propose a new component or UI pattern
+- Questions and support
+
+Please note that all contributions to Primer need to follow our code of conduct.
+
+## Participate in discussions
+
+Discussions about improvements and upcoming work happen mostly in issues and in the #design-system Slack channel. These are the repos you should watch if you’re interested in getting involved:
+
+- Design
+- Primer CSS
+- Primer Components (React)
+- Octicons
+- ...
+
+The Design Systems Team holds office hours twice a week and anyone at GitHub is free to join, ask questions and participate in ongoing conversations.
+
+## Submit an issue 
+
+If you spot something that doesn’t look right in our design, code or documentation, use one of the issue templates to submit an issue.
```

Comment:
Here we could: 
1. List all the different places for different types of issues (some will be private) - [example from Carbon](https://www.carbondesignsystem.com/how-to-contribute/bugs-and-requests)
2. Refer back to the list of repos mentioned before
3. Choose a default repo for all incoming issues

Reply:
I prefer #1. It's lengthy, but for people looking for something specific it's nice to list it out. Makes it as easy as possible to contribute.